tests/e2e/kubetest2-kops/deployer/down.go

```diff
@@ -30,6 +31,13 @@ func (d *deployer) Down() error {
 	if err := d.init(); err != nil {
 		return err
 	}
+
+	// There is no point running the rest of this function if the cluster doesn't exist
+	cluster, _ := kops.GetCluster(d.KopsBinaryPath, d.ClusterName, nil)
+	if cluster == nil {
+		return nil
```

Comment:
Would be nice to have a message, not just silent exit.
```suggestion
		klog.Infof("Cluster %q not found", d.ClusterName)
		return nil
```

Reply:
kops.GetCluster emits the warning

```
I1024 10:08:56.437000   12306 up.go:45] Cleaning up any leaked resources from previous cluster
W1024 10:08:56.910975   12306 state.go:43] failed to run /home/prow/go/src/k8s.io/kops/.build/dist/linux/amd64/kops get cluster e2e-pr16054.pull-kops-e2e-k8s-aws-calico.test-cncf-aws.k8s.io -ojson --full; stderr=Error: cluster not found "e2e-pr16054.pull-kops-e2e-k8s-aws-calico.test-cncf-aws.k8s.io"
I1024 10:08:56.913201   12306 http.go:37] curl https://ip.jsb.workers.dev
Flag --master-count has been deprecated, use --control-plane-count instead
Flag --master-size has been deprecated, use --control-plane-size instead
Flag --master-volume-size has been deprecated, use --control-plane-volume-size instead
```